CHANGELOG.md

```diff
@@ -63,21 +63,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added default sort logic to GET API calls using Read()
 
 ### Fixed
-- Fixed #3455 - Alphabetically sorting CDN Read API call [Related Github issues](https://github.com/apache/trafficcontrol/issues/3455)
```

Comment:
what happened here? did you just put them in order? OCD? :)

Reply:
Yeah, just sorted by issue number. Realized there were more. I'm gonna back it out and just have my change since at this point a clean-up of the changelog would probably be more appropriate.

Reply:
yeah maybe @ocket8888 (the release manager) can clean it up or work with you to clean it up as the last commit prior to cutting the 5.x branch.